images/atomic-registry-quickstart/registry-project-template.json

```diff
@@ -0,0 +1,130 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "registry-project-request",
+        "creationTimestamp": null
+    },
+    "objects": [
+        {
+            "kind": "Project",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "${PROJECT_NAME}",
+                "creationTimestamp": null,
+                "annotations": {
+                    "openshift.io/description": "${PROJECT_DESCRIPTION}",
+                    "openshift.io/display-name": "${PROJECT_DISPLAYNAME}",
+                    "openshift.io/requester": "${PROJECT_REQUESTING_USER}"
+                }
+            },
+            "spec": {},
+            "status": {}
+        },
+        {
+            "kind": "RoleBinding",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "registry-admin",
+                "namespace": "${PROJECT_NAME}",
+                "creationTimestamp": null
+            },
+            "userNames": [
+                "${PROJECT_ADMIN_USER}"
+            ],
+            "groupNames": null,
+            "subjects": [
+                {
+                    "kind": "User",
+                    "name": "${PROJECT_ADMIN_USER}"
+                }
+            ],
+            "roleRef": {
+                "name": "registry-admin"
+            }
+        },
+        {
+            "kind": "RoleBinding",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "system:image-pullers",
+                "namespace": "${PROJECT_NAME}",
+                "creationTimestamp": null
+            },
+            "userNames": null,
+            "groupNames": [
+                "system:serviceaccounts:${PROJECT_NAME}"
```

Comment:
This is unfortunate, you shouldn't have to do this.  I would recommend switching this binding to binding to each individual service account - that allows you to use subjects for ServiceAccount which means you don't have to have the project name.